diff --git a/frontend/src/socket.js b/frontend/src/socket.js
index c5c8ef56..4f004afc 100644
--- a/frontend/src/socket.js
+++ b/frontend/src/socket.js
@@ -1,13 +1,10 @@
 import { io } from 'socket.io-client'
-import { socketio_port } from '../../../../sites/common_site_config.json'
 import { getCachedListResource, getCachedResource } from 'frappe-ui'

 export function initSocket() {
   let host = window.location.hostname
   let siteName = window.site_name
-  let port = window.location.port ? `:${socketio_port}` : ''
-  let protocol = port ? 'http' : 'https'
-  let url = `${protocol}://${host}${port}/${siteName}`
+  let url = `https://${host}/${siteName}`

   let socket = io(url, {
     withCredentials: true,
